### ISSUE TITLE
Dependancy error with docker

### ISSUE THREAD

**Issue by neoformit** [MonashBioinformaticsPlatform/LFQ-Analyst#17]
I'm running off branch `dev_for_R4.2.1` as suggested [here](https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst/issues/16#issuecomment-1277124259).

The docker image builds ok and I can run it with `docker run -p 3838:3838 lfq-analyst`.
But when I request the page I get this error in the console:

```
[2022-10-14T02:45:14.836] [INFO] shiny-server - Error getting worker: Error: The application exited during initialization.
── Attaching packages ─────────────────────────────────────── tidyverse 1.3.2 ──
✔ ggplot2 3.3.6      ✔ purrr   0.3.4
✔ tibble  3.1.8      ✔ dplyr   1.0.10
✔ tidyr   1.2.1      ✔ stringr 1.4.1
✔ readr   2.1.2      ✔ forcats 0.5.2
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::collapse()   masks IRanges::collapse()
✖ dplyr::combine()    masks Biobase::combine(), BiocGenerics::combine()
✖ dplyr::count()      masks matrixStats::count()
✖ dplyr::desc()       masks IRanges::desc()
✖ tidyr::expand()     masks S4Vectors::expand()
✖ dplyr::filter()     masks stats::filter()
✖ dplyr::first()      masks S4Vectors::first()
✖ dplyr::lag()        masks stats::lag()
✖ ggplot2::Position() masks BiocGenerics::Position(), base::Position()
✖ purrr::reduce()     masks GenomicRanges::reduce(), IRanges::reduce()
✖ dplyr::rename()     masks S4Vectors::rename()
✖ dplyr::slice()      masks IRanges::slice()
Error in library("DEP") : there is no package called 'DEP'
Calls: runApp ... sourceUTF8 -> eval -> eval -> ..stacktraceon.. -> library
Execution halted
```

And this in the browser:
![image](https://user-images.githubusercontent.com/42562517/195751110-a15e7096-da2d-461b-8c8f-3a6a1b13cf4c.png)

**Comment by Hailey-Z** [comment]
Hi @neoformit, thanks for your feedback. I've modified the Dockerfile under the branch of `dev_for_R4.2.1`6e536bd532d97944a02b3d9645518036875a5316, please try to rebuild the image again. As an alternative, I also pushed an image to the Docker hub, you can also pull it from [here](https://hub.docker.com/repository/docker/haileyzhang/lfq-analyst) and run it on your local side. Thanks very much.

**Comment by neoformit** [comment]
Thanks for the quick reply @Hailey-Z. I rebuilt the docker image from commit https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst/commit/6e536bd532d97944a02b3d9645518036875a5316, but it still builds (successfully) with R deps errors. Perhaps the Dockerfile could be updated to fail the build when dependency installation is not successful?

I pulled `haileyzhang/lfq-analyst:1.1` and it seems to working. I should be able to use this, thanks :slightly_smiling_face:
